tools/locale.mjs

```diff
@@ -0,0 +1,78 @@
+import { readFileSync, writeFileSync } from "fs";
+import { dirname, join } from "path";
+import { fileURLToPath } from "url";
+import glob from "glob";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
```

Comment:
[__dirname](https://nodejs.org/api/modules.html#__dirname) is a built-in in Nodejs. Any reason to calculate it?

Reply:
oh, ignore me. Just learned about the mjs.